modules/backend/formwidgets/RichEditor.php

```diff
@@ -1,6 +1,7 @@
 <?php namespace Backend\FormWidgets;
 
 use App;
+use Backend\Facades\BackendAuth;
```

Comment:
No need to use the fully qualified path the BackendAuth facade, just `use BackendAuth` is fine.